### PR TITLE
Editorial Review up to IANA ConSec

### PR DESCRIPTION
**Pull request by henkbirkholz**
All changes are intended to be editorial in nature:
* removing narrative voice
* capitalization
* concretization: removing (abstract) referrals to previous sentences, such as 'this'
* added a ton of `fixme` Figure labels and incorporated them into text
  those labels till need fixing!
* nits

### CODE REVIEW

File: draft-ietf-lamps-csr-attestation.md [hunk 1]
```diff
@@ -119,30 +119,30 @@ and used in a secure environment that has controls to prevent theft or misuse".
...
-This specification thereby is intended to be as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. Instead, it focuses on (1) the conveyance of Evidence via CSRs while making minimal assumptions about content or format of the transported Evidence and (2) the conveyance of sets of certificates used for validation of Evidence.
+This specification thereby is intended to be as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. Hence, this specification focuses on (1) the conveyance of Evidence via CSRs while making minimal assumptions about content or format of the transported Evidence and (2) the conveyance of sets of certificates used for validation of Evidence.
 The certificates typically contain one or more certification paths
 rooted in a device manufacture trust anchor and the leaf certificate being
-on the device in question; the latter is the Attestation Key that signs the Evidence statement.
+on the device in question. The leaf certificate is associated with key material that takes on the role of an Attestation Key and is used to Evidence originating from the Attester.
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:134]: Should it be "end-entity certificate" rather "leaf certificate"?

File: draft-ietf-lamps-csr-attestation.md [hunk 2]
```diff
@@ -475,14 +475,15 @@ EvidenceStatementSet EVIDENCE-STATEMENT ::= {
...
 that identify them. These mappings are are used to construct
-or parse EvidenceStatements. These would typically be Evidence Statement
-formats defined in other IETF standards, defined by other standards bodies,
-or vendor proprietary formats along with the OIDs that identify them.
+or parse EvidenceStatements. Evidence Statement formats are typically
+defined in other IETF standards, other standards bodies,
+or vendor proprietary formats along with corresponding OIDs that identify them.
 
-This list is left empty in this document. However, implementers should
+This list is left undfined in this document. However, implementers can
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:486]:
```suggestion
This list is left undefined in this document. However, implementers can
```

File: draft-ietf-lamps-csr-attestation.md [hunk 3]
```diff
@@ -494,37 +495,37 @@ EvidenceStatement ::= SEQUENCE {
    hint   UTF8String OPTIONAL
 }
 ~~~
+{: #sequence-fixme title="Figure Name FixMe3"}
 
-The type is on OID indicating the format of the data contained in stmt.
+In Figure {{sequence-fixme}}, type is an OID that indicates the format of the value of stmt.
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:500]:
```suggestion
In {{sequence-fixme}}, type is an OID that indicates the format of the value of stmt.
```

File: draft-ietf-lamps-csr-attestation.md [hunk 4]
```diff
@@ -570,7 +572,7 @@ either mechanism MAY be used for carrying multiple Evidence bundles.
 
 ##  CertificateAlternatives
 
-This is an ASN.1 CHOICE construct used to represent an encoding of a
+Figure {{choice-fixme}} shows an ASN.1 CHOICE construct used to represent an encoding of a
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:575]:
```suggestion
{{choice-fixme}} shows an ASN.1 CHOICE construct used to represent an encoding of a
```

**Comment by hannestschofenig** [comment]
Are you incorporating the editorial review comments from Carl with this PR or are these your own editorial comments?

File: draft-ietf-lamps-csr-attestation.md (hunk 2 above)
Comment (henkbirkholz) [reply draft-ietf-lamps-csr-attestation.md:486]: Thanks!

File: draft-ietf-lamps-csr-attestation.md (hunk 1 above)
Comment (henkbirkholz) [reply draft-ietf-lamps-csr-attestation.md:134]: After refreshing my memory on context I'd recommend end-entity certificate. Thank you!

File: draft-ietf-lamps-csr-attestation.md (hunk 3 above)
Comment (henkbirkholz) [reply draft-ietf-lamps-csr-attestation.md:500]: Thanks. Should have re-checked the rendering

**Comment by henkbirkholz** [comment]
> Are you incorporating the editorial review comments from Carl with this PR or are these your own editorial comments?

These are my own. We went to WGLC before I found the time. So now they are review feedback to the WGLC.
I will not be able to address Carl's comments in-text, but am composing a reply to his email day per day.

File: draft-ietf-lamps-csr-attestation.md [hunk 5]
```diff
@@ -119,30 +119,30 @@ and used in a secure environment that has controls to prevent theft or misuse".
...
+This specification thereby is intended to be as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. Hence, this specification focuses on (1) the conveyance of Evidence via CSRs while making minimal assumptions about content or format of the transported Evidence and (2) the conveyance of sets of certificates used for validation of Evidence.
 The certificates typically contain one or more certification paths
-rooted in a device manufacture trust anchor and the leaf certificate being
-on the device in question; the latter is the Attestation Key that signs the Evidence statement.
+rooted in a device manufacture trust anchor and the end-entity certificate being
+on the device in question. The end-entity certificate is associated with key material that takes on the role of an Attestation Key and is used to Evidence originating from the Attester.
```
Comment (nedmsmith) [inline draft-ietf-lamps-csr-attestation.md:134]:
```suggestion
on the device in question. The end-entity certificate is associated with key material that takes on the role of an Attestation Key and is used as Evidence originating from the Attester.
```

**Review by nedmsmith — APPROVED** [review]

**Review by nedmsmith — APPROVED** [review]

File: draft-ietf-lamps-csr-attestation.md [hunk 6]
```diff
@@ -119,30 +119,30 @@ and used in a secure environment that has controls to prevent theft or misuse".
...
+This specification thereby is intended to be as technology-agnostic as it is feasible with respect to implemented remote attestation technologies. Hence, this specification focuses on (1) the conveyance of Evidence via CSRs while making minimal assumptions about content or format of the transported Evidence and (2) the conveyance of sets of certificates used for validation of Evidence.
 The certificates typically contain one or more certification paths
-rooted in a device manufacture trust anchor and the leaf certificate being
-on the device in question; the latter is the Attestation Key that signs the Evidence statement.
+rooted in a device manufacture trust anchor and the end-entity certificate being
+on the device in question. The end-entity certificate is associated with key material that takes on the role of an Attestation Key and is used as Evidence originating from the Attester.
```
Comment (ounsworth) [inline draft-ietf-lamps-csr-attestation.md:134]: Interesting. You're phrasing this as if the Evidence-signer cert is itself also evidence. Is that the intention here?

**Comment by ounsworth** [comment]
I see that you decided to keep the convention of capitalized words that refer to a RATS Architecture term; "Evidence", "Attester", etc.

**Review by ounsworth — APPROVED** [review]

**Comment by ounsworth** [comment]
I know that Henk's review is not complete, but I am merging this to `main` so that I can proceed with merging other PRs on top of it.

**Pull request merged**